test/parallel/test-repl-empty.js

```diff
@@ -7,10 +7,11 @@ const repl = require('repl');
   let evalCalledWithExpectedArgs = false;
 
   const options = {
-    eval: common.mustCall((cmd, context) => {
+    eval: common.mustCall((cmd, _context, _file, cb) => {
       // Assertions here will not cause the test to exit with an error code
       // so set a boolean that is checked later instead.
       evalCalledWithExpectedArgs = (cmd === '\n');
+      cb();
```

Comment:
Major change?

Reply:
Yes, I've updated `eval` docs to reflect this. Now it's necessary to execute `callback` command passed to `eval` in order to continue processing of next lines.

Reply:
Ok thanks, Ive marked this PR as a major change so it's only created for the next release further.